addons/web/static/src/views/fields/statusbar/statusbar_field.xml

```diff
@@ -28,7 +28,6 @@
                         class="btn o_arrow_button_current o_arrow_button disabled text-uppercase"
                         disabled="disabled"
                         role="radio"
-                        aria-label="Current state"
```

Comment:
why removing those? The values have been slightly improved in 18.0 https://github.com/odoo/odoo/commit/189a1438659fde97a67cf4ce9c81b6a38c52cdb5#diff-f688d5ae97e8bd00559c30d491252c7689da8e11b8a8a66c350d2f7deeef9b47R43

Reply:
Hi @aab-odoo,

The `aria-label` attribute is intended to provide information when it's not available by exploring text and elements, usually when the content or purpose is expressed by using visual content.

In this case, not only it's not necessary because the currently-selected status may be known thanks to the checked element, but also it overrides the status name. In other words, it's not possible anymore to know available statuses, and the `aria-label` now provides redundant information.

I wasn't aware about the change for v18.0. I think, however, the correct solution wouldn't be to improve it but to remove it, because it's not necessary. Even if it provided the necessary information, it would still be redundant because the same information is provided by the element's text and checked status.

Regards,

Reply:
@luisg123v Thanks for the explanation